sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -40,7 +40,7 @@ public async Task RecognizeHealthcareEntitiesTest()
 
             HealthcareOperation operation = await client.StartHealthcareAsync(document);
 
-            await operation.WaitForCompletionAsync();
+            await WaitForCompletionAsync(operation);
```

Comment:
I have seen some users (myself included) looking at the tests of a library to get more ideas on the behavior of it (not included in the samples) and I wonder if by having this approach it could cause some confusion.

I agree that ideally we could have something that is transparent so the dev doesn't have to remember to add this in order to speed up playback mode.
In FR we pass the [`PollingInterval`](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerLiveTestBase.cs#L14) to [`WaitForCompletionAsync()`](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs#L59) like a user will do.
It has the same problem where the dev has to remember to do this though :(